agent/installer/registry.go

```diff
@@ -57,16 +57,30 @@ func (r *registry) ListOS() (osFilter, osBundle []string) {
 	return
 }
 
-func (r *registry) ListK8s(osBundle string) []string {
-	result := make([]string, 0, len(r.osk8sInstallerMap[osBundle]))
-	for k8s := range r.osk8sInstallerMap[osBundle] {
+func (r *registry) ListK8s(osBundleHost string) []string {
+	var result []string
+
+	// os bundle
```

Comment:
Suggestion: can we update the comment to something like below.
```
// Check if the hostOS is directly supported
```

Reply:
// Check if the hostOS is directly supported
This is one possible usage.

It's just returning the k8s for an os.